### ISSUE TITLE
toast: default action hotkey can be accidentally activated due to conflicts w/ common keyboard shortcuts

### ISSUE THREAD

**Issue by jessika** [angular/material#11474]
## Bug, enhancement request, or proposal:

# CodePen and steps to reproduce the issue:
#### Code in master branch

Caused by this commit https://github.com/angular/material/commit/fedb9a33355688f5a72f5c873766a82608342d1c ("fix(toast): improve a11y support for $mdToast.simple(). improve docs")

In this commit, the toast action is now activated by default with the shortcut "ctrl+\<first letter of the toast action message\>". For all existing toasts, if the toast starts with a letter in a common keyboard shortcut (For example "v"), then "ctrl-v" will activate the toast.

#### Detailed Reproduction Steps: 
1. In JS, create a simple toast with action name that starts with a "V", for example "View"
```
$mdToast.show($mdToast.simple()
      .textContent('View the example')
      .action('View'))
.then((response) => {
  if (response == 'ok') {
    // Do "view" action
  }
});
```

2. In the browser, while the toast is on screen, try to paste something onto the screen using "ctrl-v".

### What is the expected behavior?
The toast action should not be activated.

### What is the current behavior?
The toast action is activated.

### What is the use-case or motivation for changing an existing behavior?
This change got added recently. All existing toast actions have a possibility of being activated if the user uses a keyboard shortcut that happens to activate it. The action key should have to be explicitly defined or default to a uncommonly used keyboard shortcut.

### Which versions of AngularJS, Material, OS, and browsers are affected?

- AngularJS: 
- AngularJS Material: v1.1.10 master branch
- OS: Windows, Linux
- Browsers: Chrome

**Comment by Splaktar** [comment]
That's certainly an interesting side effect. Thank you for reporting this, especially prior to the 1.1.11 release.

Requiring the shortcut to explicitly defined won't work as that will leave many apps in an inaccessible state for screen reader and keyboard users.

Perhaps we should re-consider the Control key as the default. It seems the easiest and makes the most sense, but it will also cause the most conflicts.

Control + Shift could also hit some common hotkeys like Control + Shift + V being paste unformatted.

Perhaps Control + Alt would be better default?

**Comment by jessika** [comment]
I think doing ctrl+alt+<dynamic letter> is still risky because this could still map to commonly used keybindings. For example, it would be confusing if I used ctrl+alt+l to lock the screen and the toast was activated unknowingly. Maybe the default should be a set keybinding that is uncommonly used. Another option is to have a keybinding to focus the action button, and the user still has to hit enter to press the button.

**Comment by Splaktar** [comment]
I am working on the following:
- remove the auto `actionKey` binding
- don't default an `actionHint` if `actionKey` is not defined
- generate a warning in the console if an `action` is defined without an `actionKey`
- adding more tests
